totalRP3/modules/dashboard/dashboard.lua

```diff
@@ -146,6 +146,98 @@ local ShowingHelm, ShowingCloak, ShowCloak, ShowHelm = ShowingHelm, ShowingCloak
 local Button_Cape, Button_Helmet, Button_Status, Button_RPStatus;
 local SendChatMessage, UnitIsDND, UnitIsAFK = SendChatMessage, UnitIsDND, UnitIsAFK;
 
+--- Formats a list of names for display in the About text. Each line is placed
+--  onto a separate line with a prefixing list indicator ("- ").
+--
+--  If a role is given it will be included in each name.
+local function getFormattedAboutCreditsMembers(credits, role)
+	-- No credits? We don't want to display a "- " for no reason.
+	if not credits or #credits == 0 then
+		return "";
+	end
+
+	if role then
+		-- Localise the role and then iterate over the credits, copying
+		-- each entry and including the role information.
+		local localisedRole = loc("THANK_YOU_ROLE_" .. role);
+		local creditsWithRole = Utils.table.getTempTable();
+
+		for i = 1, #credits do
+			local name = credits[i];
+			creditsWithRole[i] = loc("THANK_YOU_WITH_ROLE"):format(name, localisedRole);
+		end
+
+		credits = creditsWithRole;
+	end
+
+	-- Net result here is we'll insert a newline and a "- " between each
+	-- item in the list. We'll prefix it with a "- " in the format call so
+	-- the first item is properly indented too.
+	local formattedMembers = ("- %s"):format(table.concat(credits, "\n- "));

```

Comment:
There is a local import of `table.concat` declared at the top as `tconcat`

Reply:
I might also reformat the imports to match the convention you were using in the color picker mixing branch if you're happy with that?

Reply:
Sure. I'm trying to organize the imports in the new files to make it easier to look for stuff when we need that, but I did not have the motivation to go through the old files. I'm updating the stuff I change on the go ^^

Reply:
Oh, small side note. The IDE I am using (IntelliJ IDEA) cannot tell unused imports when there is more than one import on the same line. Those were usually made by Kaji back then. I also find them not easy to read when they are quite long. So if I was to refactor the imports I would make them all one import per line, even if it takes more space at the top of the file, for the benefit of it being easier to read and the IDE highlighting unused imports.

Reply:
Re: 1 line per import, that's more than fine with me. Considerably easier to read.